homeassistant/components/roku/select.py

```diff
@@ -18,6 +18,9 @@
 from .entity import RokuEntity
 from .helpers import format_channel_name, roku_exception_handler
 
+# Coordinator is used to centralize the data updates
```

Comment:
this comment actually only applies to read only platforms (sensor & binary sensor), since the coordinator won't parallelize the outgoing actions (e.g. switching). Is your library/device fine with potentially parallel requests here (as you set it to 0)?

Reply:
fair point trying to catch up on these new requirements. I don't feel the device would have an issue unless someone got crazy with the calls but I do think the expected behavior would be some what serial since you are controlling a device and media playback or navigation so I think 1 is likely better for the non-readonly platforms.